### ISSUE TITLE
high loss and bad performance on my own dataset

### ISSUE THREAD

**Issue by Pichairen** [qqwweee/keras-yolo3#174]
![image](https://user-images.githubusercontent.com/32973650/42679574-0561d886-86b5-11e8-8c0e-e06970f3211b.png)
The loss almost keep stable on 15, and I employ the trained_final_weights.h5 on yolo.py to test images, it performed not very well
![image](https://user-images.githubusercontent.com/32973650/42679639-41033c54-86b5-11e8-93a1-ca238b7718f3.png)

**Comment by MichalDulemba** [comment]
Similar thing happens to me. Loss stays at 16 forever. Have you fixed it?

**Comment by Pichairen** [comment]
@MichalDulemba Not yet, and it has puzzled me for many weeks

**Comment by zhitao654321** [comment]
I have the same problem. Loss stay 13.

**Comment by zhoujuanshanghai** [comment]
@Pichairen my model performs bad too ,some objects in the picture can't be detected, do u have any  solution? can i change the net structure?

**Comment by hellobuilder** [comment]
On my dataset, train loss stays at 13 and val loss stays at 33, it performed not very well at test set.

**Comment by Luffy2211** [comment]
puzzled me too ！ On my dataset,train loss =30,and val loss=31. In any case,decline train_loss,but val_loss up 30.

**Comment by Pichairen** [comment]
I think we need data augment!

**Comment by Pichairen** [comment]
@Luffy2211 @hellobuilder @zhoujuanshanghai I think we  ignore data augment！

**Comment by Luffy2211** [comment]
@Pichairen repo have data augment,like distortion,flip,scale-paste what should be usually used.And,i used them .The program has made some changes compared to the paper.I think that's the key to making a difference. The last three layers which output inference,are difference from original model.and the loss compute is difference too. It's hard to reproduce the yolov3 paper

**Comment by eain3314** [comment]
损失很大怎么办？解决了吗

**Comment by gongjunjun** [comment]
请问，最后损失高解决了吗？

**Comment by eain3314** [comment]
没有啊------------------ 原始邮件 ------------------
发件人: "gongjunjun"<notifications@github.com>
发送时间: 2019年11月20日(星期三) 下午4:26
收件人: "qqwweee/keras-yolo3"<keras-yolo3@noreply.github.com>;
抄送: "eain3314"<864770486@qq.com>;"Comment"<comment@noreply.github.com>;
主题: Re: [qqwweee/keras-yolo3] High loss and bad performance on my owndataset (#174)



请问，最后损失高解决了吗？
 
&mdash;
